config/pulumi/Pulumi.stackname.yaml.example

```diff
@@ -291,7 +291,7 @@ config:
   # Digital Ocean Managed Kubernetes
   ############################################################################
   # This is the Kubernetes version to install using Digital Ocean K8s.
-  domk8s:k8s_version: 1.22.8-do.0
+  domk8s:k8s_version: latest
```

Comment:
A good comment here might provide a link to the DO notes so users can check the release notes.

Is the actual version used written anywhere?  If we are using `latest` and I'm having issues I'd like to be able to understand the version installed and look at the release notes for it and know what other versions exist.

Reply:
I'm going to tackle this in another PR.